### ISSUE TITLE
Not compatible under OS Sierra

### ISSUE THREAD

**Issue by AdeTheux** [britg/New-Relic-Menus#12]
The App is greyed out in the Finder, and trying to open it leads to a message similar to "this App can't run on this version of OS X".

**Comment by mdesjardins** [comment]
I love this little app - sure hope you can fix it for Sierra. :(  At first I thought it was a signed/unsigned thing where Sierra no longer has the System Preferences setting that allows apps to run when they're downloaded by unsigned authors (i.e., "Anywhere"), but I've tried the workarounds for that and none seem to work.

**Comment by mdesjardins** [comment]
Not sure if this is any help at all or a red herring, but whenever I attempt to launch the app, I see a flurry of activity in the Console app related to the com.apple.CommerceKit.TransactionService before it gives up.

**Comment by britg** [comment]
@mdesjardins @AdeTheux Sorry, I haven't upgraded the machine that has this work on it to Sierra, yet. I will try to soon and give it a look.

Anyone else have a fix they're willing to commit?

**Comment by ponny** [comment]
I don't have the skills to do this so I'm paying someone via Upwork to fix it.  Expect a pull in the next few days :-)

**Comment by rodionovd** [comment]
Hi everyone! I've just tried to compile the project on Sierra and it launches just fine. Please, try this build and see if it works for you: [New Relic Menu Bars.Sierra.zip](https://github.com/britg/New-Relic-Menus/files/605698/New.Relic.Menu.Bars.Sierra.zip)

**Comment by AdeTheux** [comment]
@rodionovd So you've just compiled the current version without any changes, and it works? 🤔 Thanks for posting the ZIP though.

**Comment by rodionovd** [comment]
@AdeTheux 

> So you've just compiled the current version without any changes, and it works? 

Correct! Although I guess the Mac App Store version is a bit outdated (the last update there was in 2013)

**Comment by AdeTheux** [comment]
@britg would you be able to compile again, and update the MAS version by any chance?

**Comment by britg** [comment]
Sure, will update here when it is approved

**Comment by AdeTheux** [comment]
@britg just wondering if you submitted the app to the MAS? Or is it under review still? Thanks!

**Comment by britg** [comment]
@AdeTheux Sorry, it has not been submitted yet. The trouble is that I don't use a mac as my primary work machine anymore. I apologize for the delay. Your best bet in the mean time is to simply use this project and build the app onto your machine.

**Comment by AdeTheux** [comment]
@britg will that work if I only have a free/personal dev account?

**Comment by rodionovd** [comment]
@AdeTheux yes, it will work on your own machine just fine

**Comment by AdeTheux** [comment]
So I downloaded the files in this repo, tried to compile via Xcode but got an error 😞 I'll try @rodionovd 's version in the meantime.

**Comment by ponny** [comment]
This still isn't in the app store :\

**Comment by ponny** [comment]
Any update on this?  Still got the 2013 version on the app store.

**Comment by ponny** [comment]
Finally figured out how to get this running via XCode export.  Thanks again :-)
